### ISSUE TITLE
DOC: inconsistent documentation in spatial.SphericalVoronoi

### ISSUE THREAD

**Issue by pmla** [scipy/scipy#10484]
The documentation contains two points of confusion.

The first is a discussion of calculation of surface areas:
> The surface area of spherical polygons is calculated by decomposing them into triangles and using L’Huilier’s Theorem to calculate the spherical excess of each triangle [Weisstein]. The sum of the spherical excesses is multiplied by the square of the sphere radius to obtain the surface area of the spherical polygon. For nearly-degenerate spherical polygons an area of approximately 0 is returned by default, rather than attempting the unstable calculation.

> The reconstitution of the surface area of the sphere, measured as the sum of the surface areas of all Voronoi regions, is closest to 100 % for larger (>> 10) numbers of generators.

I cannot find any reference to this in the code though.  Am I missing something? Issue #6069 suggests that it is not implemented anywhere but I thought it best to check.

The second point of confusion is the description of 'default values' for attributes, which I don't think makes sense.  I also can't find any code which does radius estimation.

```
    radius : double
        radius of the sphere
        Default: None (forces estimation, which is less precise)
    center : double array of shape (3,)
        center of the sphere
        Default: None (assumes sphere is centered at origin)
```

#### Scipy/Numpy/Python version information:
1.3.0 and current master branch.

@tylerjereddy I think both of these issues can be solved with deletion.  I am happy to make a PR if you think that is the right solution.

**Comment by tylerjereddy** [comment]
Removing the mention of surface area is probably sensible -- I have other open PRs related to the slow process of making that available someday.

Removing the default mention for radius makes sense, since we don't actually estimate that. @niknow might have intended to do that at some point, but we didn't implement it & not sure it would make sense anymore.

For `center`, the default isn't so bad, since the assumption of placement at origin is the default I see in the code, right?

**Comment by pmla** [comment]
Thanks @tylerjereddy.  I will make these changes in a PR.

> For center, the default isn't so bad, since the assumption of placement at origin is the default I see in the code, right?

I think a description of default values for attributes is redundant, since they are handled at the parameter level.  This is the convention used in other parts of scipy, such as the `leafsize` parameter in `scipy.spatial.cKDTree`.

**Comment by pmla** [comment]
This was fixed in #10494
